UnityProject/Assets/Scripts/Health/Living/PlayerHealth.cs

```diff
@@ -219,54 +219,7 @@ protected override void OnDeathActions()
 				PlayerMove pm = gameObject.GetComponent<PlayerMove>();
 
 				ConnectedPlayer player = PlayerList.Instance.Get(gameObject);
-				
-				string killerName = "Stressful work";
-				if (LastDamagedBy != null)
-				{
-					killerName = PlayerList.Instance.Get(LastDamagedBy).Name;
-				}
 
-				if (killerName == player.Name)
-				{
-					PostToChatMessage.Send(player.Name + " commited suicide", ChatChannel.System); //Killfeed
-				}
-				else if (killerName.EndsWith(player.Name))
-				{
-					// chain reactions
-					PostToChatMessage.Send(
-						player.Name + " screwed himself up with some help (" + killerName + ")",
-						ChatChannel.System); //Killfeed
-				}
-				else
-				{
-					PlayerList.Instance.UpdateKillScore(LastDamagedBy, gameObject);
-
-					string departmentKillText = "";
-					if (LastDamagedBy != null)
-					{
-						JobDepartment killerDepartment =
-							SpawnPoint.GetJobDepartment(LastDamagedBy.GetComponent<PlayerScript>().JobType);
-						JobDepartment victimDepartment =
-							SpawnPoint.GetJobDepartment(gameObject.GetComponent<PlayerScript>().JobType);
-
-						if (killerDepartment == victimDepartment)
-						{
-							departmentKillText = ", losing " + killerDepartment.GetDescription() +
-							                     " 1 point for team killing!";
-						}
-						else

```

Comment:
There doesn't seem to be any attempt here to attribute a kill to either NukeOps or the Station crew. Instead you have just removed all the reporting code